pulp_service/pulp_service/app/models.py

```diff
@@ -164,3 +165,12 @@ class Meta:
                 "Can manage role assignments on Feature ContentGuard",
             ),
         )
+
+
+class ArtifactVulnerability(models.Model):
+    """
+    Model used in content vulnerability scan
```

Comment:
```suggestion
    Model used in vulnerability report.
```

Similar replacement of `scan` with `report` throughout the PR.